### DISCUSSION TITLE
issues

### DISCUSSION THREAD

**Discussion by ngwansalah** [community/community#67809] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

hello 
I have issues with my account

**Comment by cehfisher** [comment]
Hi @ngwansalah - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @ngwansalah ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
